app/Http/Controllers/ManagerProfileController.php

```diff
@@ -65,13 +65,15 @@ public function show(Request $request, Manager $manager)
             ]
         ];
 
+        $noInfo = $manager_profile['no_info'];
 
         return view('applicant/manager', [
             'manager_profile' => $manager_profile,
             'urls' => Lang::get('common/urls'),
             'manager' => $manager,
             'manager_profile_photo_url' => '/images/user.png', // TODO get real photo.
             'manager_profile_sections' => $manager_profile_sections,
+            'noInfo' => $noInfo,
```

Comment:
In the future I'd just 
```php
'no_info' => $manager_profile['no_info'],
```
but it's not hurting anything with the extra variable declaration

Reply:
Okay will do